app/helpers/physical_storage_consumer_helper/textual_summary.rb

```diff
@@ -0,0 +1,58 @@
+module PhysicalStorageConsumerHelper::TextualSummary
+  include TextualMixins::TextualGroupTags
+
+  #
+  # Groups
+  #
+
+  def textual_group_properties
+    TextualGroup.new(
+      _("Properties"),
+      %i[
+        name
+        ems
+        physical_storage
+      ]
+    )
+  end
+
+  def textual_group_relationships
+  end
```

Comment:
Is this meant to be empty? We could minimally do this
```
def textual_group_relationships
    TextualGroup.new(_("Relationships"), %i[ems])
end
```

Reply:
removed